Most Obex Push receivers are broken when getting a path inside
the push. We will change the working directory.
This patch is included upstream already

Index: obex.cc
===================================================================
RCS file: /cvs/opie/core/obex/obex.cc,v
retrieving revision 1.9
diff -u -r1.9 obex.cc
--- obex.cc	12 Sep 2004 18:55:56 -0000	1.9
+++ obex.cc	29 Mar 2005 23:28:37 -0000
@@ -74,8 +74,10 @@
     }
     // OProcess inititialisation
     m_send = new OProcess();
+    m_send->setWorkingDirectory( QFileInfo(m_file).dirPath(true) );
+
     *m_send << "irobex_palm3";
-    *m_send << QFile::encodeName(m_file);
+    *m_send << QFile::encodeName(QFileInfo(m_file).fileName());
 
     // connect to slots Exited and and StdOut
     connect(m_send,  SIGNAL(processExited(Opie::Core::OProcess*) ),
